source/administration/config-settings.rst

```diff
@@ -224,21 +271,21 @@ Authentication settings to enable account creation and sign in with email, GitLa
 
 Email
 ``````````````````````````
-Allow Sign Up With Email ``"EnableSignUpWithEmail": true``
+Enable account creation with email ``"EnableSignUpWithEmail": true``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
 
 ``true``: Allow team creation and account signup using email and password.
 
 ``false``: Email signup is disabled and users are not able to invite new members. This limits signup to single-sign-on services like OAuth or LDAP.  
 
-Allow Sign In With Email ``"EnableSignInWithEmail": true``
+Enable sign-in with email ``"EnableSignInWithEmail": true``
```

Comment:
It's in the system console, so maybe only fixed in v3.3, but `sign-in`--> `sign in`